src/delegate.py

```diff
@@ -104,12 +104,19 @@ def handler(event, context: LambdaContext) -> dict:
             if not domains.issubset(refresh_token['domains']):
                 return bad_request('', 'domain requested outside refresh_token')
 
+        # Validate no commas in subject or existing sub chain to avoid join ambiguity
+        if ',' in subject:
+            return bad_request('', 'subject contains invalid comma')
+        existing_sub = refresh_token.get('sub', [])
+        if any(',' in s for s in existing_sub if isinstance(s, str)):
+            return bad_request('', 'existing sub chain contains invalid comma')
```

Comment:
als existing_sub een string is gaat dit problemen geven denk ik 
`[s for s in "foo""` is `["f", "o", "o"]` IIRC